diff --git a/src/Misc/Bank.cpp b/src/Misc/Bank.cpp
index f6d87f9..2d79e6b 100644
--- a/src/Misc/Bank.cpp
+++ b/src/Misc/Bank.cpp
@@ -1532,9 +1532,9 @@ bool Bank::establishBanks(optional<string> bankFile)
     string bankdirs[] = {
         foundLocal + "yoshimi/banks",
         "/usr/share/yoshimi/banks",
-        "/usr/local/share/yoshimi/banks",
+        "/app/extensions/Plugins/Yoshimi/share/yoshimi/banks",
         "/usr/share/zynaddsubfx/banks",
-        "/usr/local/share/zynaddsubfx/banks",
+        "/app/extensions/Plugins/ZynFusion/share/zynaddsubfx/banks",
         foundLocal + "zynaddsubfx/banks",
         extendLocalPath("/banks"),
         "@end"
diff --git a/src/Misc/Config.cpp b/src/Misc/Config.cpp
index 1bb5e43..3d92391 100644
--- a/src/Misc/Config.cpp
+++ b/src/Misc/Config.cpp
@@ -723,7 +723,7 @@ void Config::defaultPresets()
          * We shouldn't be setting these directly
          */
         "/usr/share/yoshimi/presets",
-        "/usr/local/share/yoshimi/presets",
+        "/app/extensions/Plugins/Yoshimi/share/yoshimi/presets",
         "@end"
     };
     int i = 0;
