diff --git a/src/Misc/Bank.cpp b/src/Misc/Bank.cpp
index ae1be36..91d78ee 100644
--- a/src/Misc/Bank.cpp
+++ b/src/Misc/Bank.cpp
@@ -1580,9 +1580,9 @@ bool Bank::establishBanks(optional<string> bankFile)
     string bankdirs[] = {
         foundLocal + "yoshimi/banks",
         "/usr/share/yoshimi/banks",
-        "/usr/local/share/yoshimi/banks",
+        "/app/extensions/Plugins/Yoshimi/share/yoshimi/banks",
         "/usr/share/zynaddsubfx/banks",
-        "/usr/local/share/zynaddsubfx/banks",
+        "/app/extensions/Plugins/ZynFusion/share/zynaddsubfx/banks",
         foundLocal + "zynaddsubfx/banks",
         extendLocalPath("/banks"),
         "@end"
diff --git a/src/Misc/Config.cpp b/src/Misc/Config.cpp
index c25ee3c..e2e4537 100644
--- a/src/Misc/Config.cpp
+++ b/src/Misc/Config.cpp
@@ -1268,7 +1268,7 @@ void Config::defaultPresets()
                                          * We shouldn't be setting these directly
                                          */
                                      ,string{"/usr/share/yoshimi/presets"}
-                                     ,string{"/usr/local/share/yoshimi/presets"}
+                                     ,string{"/app/extensions/Plugins/Yoshimi/yoshimi/presets"}
                                      };
     int actual{0};
     Log("Setup default preset directories...", _SYS_::LogNotSerious);
